frb_codegen/src/utils.rs

```diff
@@ -42,6 +42,80 @@ where
         .collect::<Vec<_>>()
 }
 
+// https://dart.dev/guides/language/language-tour#keywords
+const DART_KEYWORDS: [&str; 63] = [
+    "abstract",
+    "else",
+    "import",
+    "show",
+    "as",
+    "enum",
+    "in",
+    "static",
+    "assert",
+    "export",
+    "interface",
+    "super",
+    "async",
+    "extends",
+    "is",
+    "switch",
+    "await",
+    "extension",
+    "late",
+    "sync",
+    "break",
+    "external",
+    "library",
+    "this",
+    "case",
+    "factory",
+    "mixin",
+    "throw",
+    "catch",
+    "false",
+    "new",
+    "true",
+    "class",
+    "final",
+    "null",
+    "try",
+    "const",
+    "finally",
+    "on",
+    "typedef",
+    "continue",
+    "for",
+    "operator",
+    "var",
+    "covariant",
+    "Function",
+    "part",
+    "void",
+    "default",
+    "get",
+    "required",
+    "while",
+    "deferred",
+    "hide",
+    "rethrow",
+    "with",
+    "do",
+    "if",
+    "return",
+    "yield",
+    "dynamic",
+    "implements",
+    "set",
+];
+
+fn check_for_keywords(v: &[String]) -> anyhow::Result<()> {
```

Comment:
IMO it isn't worth a function, that's 3 lines.

Reply:
Hmm, I meant the function `check_for_keywords`, IDK what went wrong with selection :)

Reply:
well, making it a function is indeed like adding a comment (via function name) :)